src/modules/registration/registration.module.ts

```diff
@@ -0,0 +1,77 @@
+import { Module } from '@nestjs/common';
+import { LoginService } from '@app/modules/registration/services/login.service';
+import { LoginResolver } from '@app/modules/registration/resolvers/login.resolver';
+import { TokensService } from '@app/modules/registration/services/tokens.service';
+import { UsersModule } from '@app/modules/users/users.module';
+import { OTPService } from './services/otp.service';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { JwtModule } from '@nestjs/jwt';
+import { AppPrismaModule } from '@app/app-prisma/app-prisma.module';
+import { PassportModule } from '@nestjs/passport';
+import { JwtStrategy } from './strategies/jwt.strategy';
+import { JwtGuard } from './guards/jwt.guard';
+import { TwilioModule } from 'nestjs-twilio';
+import { JwtRefreshGuard } from './guards/jwt-refresh.guard';
+import { JwtRefreshTokenStrategy } from './strategies/jwt-refresh-token.strategy';
+import {
+  IAppConfig,
+  IAuthentication,
+  ITwilio,
+} from '@app/app-config/app-config.interface';
+
+@Module({
+  imports: [
+    UsersModule,
+    PassportModule.register({ defaultStrategy: 'jwt', property: 'user' }),
+    AppPrismaModule,
+    JwtModule.registerAsync({
+      imports: [],
+      useFactory: async (configService: ConfigService<IAppConfig>) => ({
+        publicKey: Buffer.from(
```

Comment:
Why we need `publicKey` and `privateKey`?

Reply:
using RS256 Signing Algorithm [RS256 vs HS256](https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/)

Reply:
@falyoun